### PR TITLE
formatted help

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: Command/CreateDatabaseDoctrineCommand.php [hunk 1]
```diff
@@ -34,19 +34,17 @@ protected function configure()
...
         $this
             ->setName('doctrine:database:create')
-            ->setDescription('Creates the configured databases')
+            ->setDescription('Creates the configured database')
             ->addOption('connection', null, InputOption::VALUE_OPTIONAL, 'The connection to use for this command')
             ->addOption('if-not-exists', null, InputOption::VALUE_NONE, 'Don\'t trigger an error, when the database already exists')
             ->setHelp(<<<EOT
-The <info>doctrine:database:create</info> command creates the default
-connections database:
+The <info>doctrine:database:create</info> command creates the default connections database:
```
Comment (stof) [inline Command/CreateDatabaseDoctrineCommand.php]: Can you also switch this to use ``%command.name%`` for the command name (and ``%command.full_name%`` on next line for ``app/console doctrine:database:create``) ? It would avoid hardcoding the name multiple times

Comment (guilhermeblanco) [inline Command/CreateDatabaseDoctrineCommand.php]: @stof too late... it'll have to be another PR! =P

Comment (OskarStark) [inline Command/CreateDatabaseDoctrineCommand.php]: i'll do

Comment (OskarStark) [inline Command/CreateDatabaseDoctrineCommand.php]: #427 
ping @stof @guilhermeblanco
